teuthology/task/install/__init__.py

```diff
@@ -198,15 +198,17 @@ def get_package_list(ctx, config):
         debs = filter(lambda p: not p.endswith('-dbg'), debs)
         rpms = filter(lambda p: not p.endswith('-debuginfo'), rpms)
 
-    excluded_packages = config.get('exclude_packages', list())
-    if excluded_packages:
-        log.debug("Excluding packages: {}".format(excluded_packages))
-
-        def exclude(pkgs):
-            return list(set(pkgs).difference(set(excluded_packages)))
+    def exclude(pkgs, exclude_list):
+        return list(set(pkgs).difference(set(exclude_list)))
```

Comment:
might want to update 

https://github.com/ceph/teuthology/blob/da24833932b65ee8d660a6b9452a9cc33fee1c91/teuthology/test/task/test_install.py#L43-L45

accordingly, as the order of elements in `list(set(["foo", "bar"]))` is determined by how python hashes string.